index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello World!we are software developers</h1>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>Welcome</h1>
+    </section>
+    <section id="projects">
+      <div class="project-tile"> </div>
+      <a href="/project-1">Project 1</a>
+    </section>
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome Section</a>
+    </nav>
+    <a id="profile-link" target="_blank" href="https://www.github.com/munsa1" >GitHub</a>
+
+    <style>
+      @media only screen and (max-width: 500px){
+        section {
+          font-size: 40px;
+        }
+      }
+      #welcome-section {
+        height: 100vh;
+      }
+      nav{
+        position: fixed;
+        top: 0;
+
+      }
```

Comment:
Always remember,
it is best practice to not leave spaces after your code inside curly brackets as is the case here.
please revert to the linter checks to rectify these tedious errors. 😁